pkg/minikube/cluster/cluster.go

```diff
@@ -292,29 +301,37 @@ func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 
 	switch config.VMDriver {
 	case "virtualbox":
-		driver = createVirtualboxHost(config)
+		driverOptions, err = createVirtualboxHost(config)
```

Comment:
@hferentschik , again commenting here for the same suggestion as I did before.

Now, `createVirtualboxHost` is of no real meaning here since we are only trying to get desired `driverOptions` from `config`.
So, it would be great if we rename to it something like `getVirtualboxDriverOptions` .